plutus-core-spec/main.tex

```diff
@@ -875,6 +888,18 @@ \section{Syntax}
 
 \subfile{figures/PlutusCoreGrammar}
 
+%%%%%%%
+\red{
+Some comments on the syntax...
+\begin{itemize}
+\item Are Values also Terms?  If not, then I don't think $\wrap{\alpha}{S}{V}$ is a term, for example (perhaps this is correct).
+%
+\item Under Term (and maybe also under Value), should $\abs{\alpha}{K}{V}$ be $\abs{\alpha}{K}{M}$?  
```

Comment:
Someone noticed this problem after trying to type check `/\(a :: *) -> error a`. Do not know whether there is a consensus already.